app/src/main/res/layout/sensor_main.xml

```diff
@@ -3,8 +3,20 @@
     android:id="@+id/layout_container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fitsSystemWindows="true"
-    android:orientation="vertical">
+    android:orientation="vertical"
+
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <android.support.v7.widget.Toolbar
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:id="@+id/toolbar"
+        android:background="?attr/colorPrimary"
+        android:minHeight="?attr/actionBarSize"
+        android:fitsSystemWindows="true"
+        app:title="Sensor QuickView"
+        app:titleTextColor="@color/white"
+        app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"></android.support.v7.widget.Toolbar>
```

Comment:
`</android.support.v7.widget.Toolbar>` part is not necessary as you can close the tag with `app:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar" />`

Reply:
Please fix that in all the occurrences in this PR!